### DISCUSSION TITLE
第4节课作业(17班)

### DISCUSSION THREAD

**Discussion by Jianfeng777** [InternLM/Tutorial#276] [作业]
请大家将第四节课的作业上传到这里哦！

**Comment by l241025097** [comment]
第四节课作业
[04XTuner大模型单卡低成本微调实战.pdf](https://github.com/InternLM/tutorial/files/13930044/04XTuner.pdf)

**Comment by futtttttttttttt** [comment]
https://zhuanlan.zhihu.com/p/677779844

**Comment by StarDuster** [comment]
https://gist.github.com/StarDuster/7dcd38e478725aff64891c93dc6d9618

**Comment by zcg-joker** [comment]
[https://blog.csdn.net/m0_55764641/article/details/135589334](https://blog.csdn.net/m0_55764641/article/details/135589334)

**Comment by fine-lab** [comment]
https://blog.csdn.net/SmallZCan_t/article/details/135589578?spm=1001.2014.3001.5501

**Comment by AxrLH** [comment]
第四课作业
https://blog.csdn.net/Axr_Samuel/article/details/135586657?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135586657%22%2C%22source%22%3A%22Axr_Samuel%22%7D

**Comment by Alisa742** [comment]
https://blog.csdn.net/weixin_53222965/article/details/135590663?spm=1001.2014.3001.5502
